CONTRIBUTING.md

```diff
@@ -187,18 +185,18 @@ The PR title must follow this format:
 
 
 ```
-<type>: <subject>
+<type>(<scope>): <subject>
```

Comment:
did we not agree on `<type>: (<Scope>) <subject>` ? Even the PR description seems to have swapped the `Before` and `Now` parts.

Reply:
It was discussed with @InnaAtanasova to use Angular commit message format - `<type>(<scope>): <subject>`

Reply:
I see. And this won't cause any issues in the release notes? 
So the `scope` is no longer capitalized?

Reply:
I thought scope needed to be to the right of the `:` so it will appear in the release notes `<type>: (<scope>) <subject>`

Reply:
@Seamoo13 
we have situation that part of our commits in the repository to use one commit format type(scope): message and we use another format like type: (Scope) message

and that’s why we decided to use one commit message format